lib/src/misc/extensions.dart

```diff
@@ -3,62 +3,34 @@ import 'dart:ui';
 
 /// Extension methods for the math.[Point] class
 extension PointExtension<T extends num> on Point<T> {
```

Comment:
We should deprecate this entire extension method and all methods/operators within, or mark it as internal at least.